cob_hardware_config/cob4-1/config/sensorring_driver.yaml

```diff
@@ -2,9 +2,11 @@ name: sensorring_controller
 
 defaults:
   eds_pkg: cob_hardware_config
-  eds_file: "cob4-1/config/Elmo.dcf"
+  eds_file: "cob4-2/config/Elmo.dcf"
```

Comment:
why cob4-2? is it different? the Elmo.dcf should be a common file , and the cobX-Y/config/Elmo.dcf just a link